### PR TITLE
Final Capstone: Doctor's Appointment

### PR DESCRIPTION
**Pull request by arnoldnekemiah**
In this project We did the following;

- Built the frontend with react and redux.
- Add components to perform various actions based on the project requirements.
- Add responsiveness to the app.
- Fetched data from the backend api.

Backend PR link: (https://github.com/agneta1991/book-an-appointment-capstone/pull/39)

### CODE REVIEW

File: src/featureSlice/imageSlice.js
Comment (Meltrust) [inline src/featureSlice/imageSlice.js:1]: - [x] This file is full of commented code.  Kindly remove it if is not being used. 👍

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import MyReservations from './components/MyReservations/MyReservations';
+import ReservationForm from './containers/reservationform/reservationform';
+import ProtectedRoute from './components/ProtectedRoute';
+import Footer from './components/Footer/Footer';
+import Dashboard from './components/Dashboard/Dashboard';
+import DoctorInfo from './components/Feature/DoctorInfo/DoctorInfo';
+import DoctorForm from './components/DoctorForm/DoctorForm';
+import './app.css';
+
+function App() {
```
Comment (Meltrust) [inline src/App.js:24]: - [x]  Please, kindly refactor this main App functional component to ES6 arrow function, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

- [ ] OPTIONAL: Refactor the rest of the functions.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry:

image


That way, your code will meet the latest standards. 💪

Comment (Meltrust) [inline src/App.js:24]: - [ ] In the mobile view, the content is not fitting the screen:

image

image

It seems that the nav is getting in the way of the content.

Kindly, touch up your styling so the content fits the screen in the mobile view. 👍

Comment (Meltrust) [inline src/App.js:24]: - [ ] I cannot login as any of the included admins, or even if I try to create a new one through the console:

image

image

image


Kindly, make sure admins can login, so we can test the add and delete functionality on your project. 👍

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi,

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation, but you are almost there!






### To highlight:




#### You are really close to finishing the Microverse program!! Keep it up! 👍👍👍

You can do it

### After implementing the requested changes, please submit another review request. ♻️
_Check the comments under the review._







Cheers and Happy coding!👏👏👏

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the previous reviews unless it is requested otherwise._**

------

File: src/App.js (hunk 1 above)
Comment (agneta1991) [reply src/App.js:24]: mobile version css has been added, the screen has to be max at 375 width, please check the measurements when checking mobile version

Comment (agneta1991) [reply src/App.js:24]: the log in is a user name as per requirements and password not email and password.

File: src/app.css [hunk 2]
```diff
@@ -0,0 +1,28 @@
+.main-layout {
```
Comment (Meltrust) [inline src/app.css:1]: - [ ] OPTIONAL: For the mobile view, the screen width of 375px is very tiny.  I would suggest making it bigger, so the majority of phones can enjoy your work.

**Review by Meltrust — APPROVED** [review]
# **Wow, you did it 🎉**
Brilliant
## Thank you for the changes implemented 💪 🥇 ㊗️ 
#### _Unless you want to add more features, go ahead to your final presentation ⏩ ⏩ ⏩_ 
## You are about to finish the Microverse program.  You have come a long way!!!




### Good luck in the software industry!!  I'll see you there. ✨
### **Congratulations!!!!!!**  🎉
applause






Cheers and Happy coding!👏👏👏


------

**Pull request merged**